pkg/orchestrator/gitHubActions.go

```diff
@@ -94,11 +131,227 @@ func (g *GitHubActionsConfigProvider) GetPullRequestConfig() PullRequestConfig {
 }
 
 func (g *GitHubActionsConfigProvider) IsPullRequest() bool {
-	_, exists := os.LookupEnv("GITHUB_HEAD_REF")
-	return exists
+	return getEnv("GITHUB_HEAD_REF", "") != ""
 }
 
 func isGitHubActions() bool {
-	envVars := []string{"GITHUB_ACTION", "GITHUB_ACTIONS"}
-	return areIndicatingEnvVarsSet(envVars)
+	return getEnv("GITHUB_ACTIONS", "") == "true"
+}
+
+type run struct {
```

Comment:
I think it would be useful to add a comment about where this comes from (a GitHub API JSON response schema I assume)